cmd/system/node.go

```diff
@@ -0,0 +1,138 @@
+package system
+
+import (
+	"fmt"
+	"github.com/alexellis/arkade/pkg/archive"
+	"github.com/alexellis/arkade/pkg/env"
+	"github.com/alexellis/arkade/pkg/get"
+	cp "github.com/otiai10/copy"
+	"github.com/spf13/cobra"
+	"io"
+	"net/http"
+	"os"
+	"regexp"
+	"strings"
+)
+
+func getLatestVersion(version, channel string) (*string, error) {
+	resp, err := http.Get(fmt.Sprintf("https://nodejs.org/download/%s/%s", channel, version))
+	if err != nil {
+		return nil, err
+	}
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return nil, err
+	}
+	regex := regexp.MustCompile(`(?m)node-(.*)-linux-.*"`)
+	result := regex.FindStringSubmatch(string(body))
+	if len(result) < 2 {
+		return nil, fmt.Errorf("could not find latest version for %s", version)
+	}
+	return &result[1], nil
+}
+
+func MakeInstallNode() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "node",
+		Short: "Install Node.js",
+		Long:  `Install the Node.js JavaScript runtime ✨🐢🚀✨`,
+		Example: `arkade system install node
+  arkade system install node --version v17.9.0`,
```

Comment:
Does `--version` also support `lts`? Given the layout of the website, I could see this as a common request.

Reply:
Nope, currently not. Should I add lts as alias too? Would mean some changes as i need to evaluate the json too.